internal/config/config.go

```diff
@@ -32,6 +32,10 @@ import (
 )
 
 var (
+	// Default sections
+	sectionDefaults = []string{"agent", "global_tags", "output",
+		"processor", "aggregator", "input"}
```

Comment:
I think maybe we should switch the plugin sections to be plural: `--section-filter inputs:outputs:processors:aggregators`.